cmd/connector_celo.go

```diff
@@ -28,6 +28,21 @@ func connectChain(
 		)
 	}
 
+	// DEPRECATED: config.Celo.ContractAddresses is the correct container
+	// for the TBTCSystem address from now on; default to Extensions.TBTC and
+	// warn if the ContractAddresses version is not set yet.
+	_, exists := config.Celo.ContractAddresses[celo.TBTCSystemContractName]
+	if !exists && len(config.Extensions.TBTC.TBTCSystem) != 0 {
+		logger.Warn(
+			"TBTCSystem address configuration in Extensions.TBTC.TBTCSystem " +
+				"is DEPRECATED and will be removed. Please configure the " +
+				"TBTCSystem address alongside BondedECDSAKeep under " +
+				"Celo.ContractAddresses.",
+		)
+		config.Celo.ContractAddresses[celo.TBTCSystemContractName] =
+			config.Extensions.TBTC.TBTCSystem
+	}
```

Comment:
Since we don't have any live users of the Celo integration, I don't know if we need to be concerned with backwards compatibility there. Thoughts on dropping this bit entirely?

Reply:
Sure, done in https://github.com/keep-network/keep-ecdsa/commit/27e91c2599745263ffca647ed1a25ff7782a24e9